bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActions.java

```diff
@@ -115,10 +116,11 @@ public class EnergiDataServiceActions implements ThingActions {
     }
 
     @RuleAction(label = "@text/action.calculate-cheapest-period.label", description = "@text/action.calculate-cheapest-period.description")
-    public @ActionOutput(name = "result", type = "java.util.Map<String, Object>") Map<String, Object> calculateCheapestPeriod(
-            @ActionInput(name = "earliestStart", type = "java.time.Instant") Instant earliestStart,
-            @ActionInput(name = "latestEnd", type = "java.time.Instant") Instant latestEnd,
-            @ActionInput(name = "duration", type = "java.time.Duration") Duration duration) {
+    public @ActionOutputs({ @ActionOutput(name = "CheapestStart", label = "Cheapest Start", type = "java.time.Instant"),
```

Comment:
Does core properly serialise Instant and return it from the UI?

Reply:
I just tested with Instant as input and output:
```
    @RuleAction(label = "test2", description = "Test action 2")
    public @ActionOutputs({ @ActionOutput(name = "result", label = "Test result", type = "java.lang.Integer"),
            @ActionOutput(name = "instant", label = "Instant", type = "java.time.Instant") }) Map<String, Object> testAction2(
                    @ActionInput(name = "instantParam", label = "instantParam parameter", description = "Descr instantParam parameter") Instant instantParam) {
        logger.info("testAction2 {}", instantParam);
        return Map.of("result", "The value returned", "instant", instantParam.plus(5, ChronoUnit.DAYS));
    }
```
and UI shows an object:
![image](https://github.com/user-attachments/assets/f430d5df-37f1-431e-9b89-1cdf15fda334)
Not ideal.

Reply:
This should be fixable so the UI instead shows that `{ ... }`.